pkg/webhook/server/server.go

```diff
@@ -137,15 +138,15 @@ func (s *Server) waitForTLSReadiness() error {
 	})
 }
 
-func (s *Server) Start(stop <-chan struct{}) error {
+func (s *Server) Start(ctx context.Context) error {
 	s.log.Info("Starting nodenetworkconfigurationpolicy webhook server")
 
 	err := s.waitForTLSReadiness()
 	if err != nil {
 		return errors.Wrap(err, "failed watting for ready TLS key/cert")
 	}
 
-	err = s.webhookServer.Start(stop)
+	err = s.webhookServer.Start(ctx)
```

Comment:
so you can now add a context cancel function instead of the stop channel? (just asking, not suggesting you need to add such a function)

Reply:
You must, they hare replace one for another, after context concept appeart at golang is the most idiomatic way of do the stop ching.